src/lib-components/CardMeta.vue

```diff
@@ -107,10 +129,16 @@ const parsedLocations = computed(() => {
     }
   })
 })
+
+// if we want to use theme pattern based on passing classes prop to each component
+const parsedClasses = computed(() => {
```

Comment:
Rename parsedClasses to classes, like other components toplevel computed class method